frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Move needed information to the development README file. And we can possibly remove this one.

Reply:
This readme has been deleted, information moved to development readme.